en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -186,29 +186,28 @@ all_videos <- read_csv(video_files, col_names = FALSE, id = "videoId", show_col_
   select(videoId, videoChannelTitle = channelTitle, videoTitle = title, commentCount)
 ```
 
-To confirm channel titles, and the number of comments per channel, simply input `all_videos` to print the results. Finally, run the following code to join the video and comment data:
+To confirm channel titles and the number of comments per channel, simply input `all_videos` to print the results. Finally, run the following code to join the video and comment data:
 
 ```
 all_data <- inner_join(all_comments, all_videos)
 count(all_data, sort(videoChannelTitle))
 ```
 
-Alternatively, if you would like to utilize our sample data, you can download it from the Github repository. Use this code to read in the sample data: 
+Alternatively, if you are using our sample data, you can read it in with the following code: 
 
 ```
 all_data <- read.csv("ytdt_data/all_data.csv")

```

Comment:
Could you double check the file referred to here as well? I'm not sure where I would find `ytdt_data/all_data.csv` if I had downloaded `AllComments__News_v_Comedy_BLM_Coverage.xlsx` as instructed in the note above.

Reply:
Pretty sure the AllComments is a very old name that shouldn't still be there. I need to double check what that filename should be